src/bundle/ui-dev/src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -508,16 +508,13 @@ export const fetchAdminConfig = async ({ token, siteaccess, accessToken, instanc
     return adminUiConfig.ApplicationConfig;
 };
 
-export const findSuggestions = (
-    { siteaccess, token, parentLocationId, accessToken, instanceUrl = DEFAULT_INSTANCE_URL, limit = QUERY_LIMIT, offset = 0 },
-    callback,
-) => {
+export const findSuggestions = ({ siteaccess, token, parentLocationId, accessToken, instanceUrl = DEFAULT_INSTANCE_URL }, callback) => {
     const body = {
         ViewInput: {
             identifier: 'view_with_aggregation',
             LocationQuery: {
-                limit,
-                offset,
+                limit: 0,
```

Comment:
why limit 0 and how will it work? load all?

Reply:
we want to avoid loading contents for performance reasons but the aggregation itself with still be done giving us information on content type numbers

Reply:
Normally I'm against comments in the code but here it would be nice to have this explained.

Reply:
Comment added